opm/material/fluidmatrixinteractions/BrooksCoreyParams.hpp

```diff
@@ -42,19 +43,19 @@ namespace Opm {
  *\see BrooksCorey
  */
 template <class TraitsT>
-class BrooksCoreyParams
+class BrooksCoreyParams : public AssertFinalized
 {
     typedef typename TraitsT::Scalar Scalar;
-
+protected:
+    using AssertFinalized :: assertFinalized_;
```

Comment:
I think I'd prefer to avoid this `using` and use `AssertFinalized::ensure()` (or similar) at the calling sides.